src/EnergyPlus/AirflowNetworkBalanceManager.hh

```diff
@@ -232,6 +233,63 @@ namespace AirflowNetworkBalanceManager {
 	Real64
 	GetZoneInfilAirChangeRate( int const ZoneNum ); // hybrid ventilation system controlled zone number
 
+	// derived class or struct
+	struct OccupantVentilationControlProp {
+
```

Comment:
@lgu1234 In my opinion it would be better to add real enumerations here. Something like this:

    enum class OpenStatus { FreeOperation, MinCheckForceOpen, MinCheckForceClose };
    enum class OpenCloseStatus { FreeOperation, ForceChange, NoAction };

Then OpeningStatus is an "OpenStatus" type, OpeningProbStatus and ClosingProbStatus are the "OpenCloseStatus" type. I couldn't come up with better names.